mxcube3/ui/components/SampleQueue/CharacterisationTaskItem.jsx

```diff
@@ -172,7 +172,7 @@ export default class TaskItem extends Component {
       <tr>
         <td><a>{parameters.osc_start.toFixed(2)}</a></td>
         <td><a>{parameters.osc_range.toFixed(2)}</a></td>
-        <td><a>{parameters.exp_time.toFixed(2)}</a></td>
+        <td><a>{parameters.exp_time.toFixed(3)}</a></td>
```

Comment:
I smiled at this line, they made me put 4 here. No problem, 3 is fine

Reply:
Okey, this could perhaps become configurable, although we see little need for that at the moment. I think there is more or less an agreement on the precision needed for these values. So good if 3 is fine for you